docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,182 @@
+| Aanleiding     | Een consumer wil een lijst van alle beschikbare CATALOGUSsen of een specifieke catalogus opvragen.         |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Raadplegen*   |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | Een lijst met beschikbare catalogussen wordt teruggegeven aan de consumer bij het opvragen van de lijst.<br>Een specificieke catalogus wordt teruggegeven aan de consumer. |
+| Gegevens       | objecttype CATALOGUS inclusief een lijst met relaties naar daarin gedefinieerde ZAAKTYPEn.<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/catalogus                                                                                                   |
+| Samenhang      | nvt. |
+| Bijzonderheden | nvt. |
+        
+
+
+### Resource: zaaktype
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil (een lijst van beschikbare) zaaktype informatie opvragen uit de catalogus.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Raadplegen*   |      |
+| - voorwaarde   | Om zaaktype informatie op te kunnen vragen moet de consumer de UUID van de catalogus waarin het zaaktype is opgenomen weten. Deze UUID kan opgevraagd worden met de catalogus bevragingen services.       |
+| - gevolg       | Een lijst met in de catalogus opgenomen zaaktypen wordt teruggegeven aan de consumer bij het opvragen van de lijst.<br>Een specificiek zaaktype wordt teruggegeven aan de consumer.         |
+| Gegevens       | ZAAKTYPE inclusief STATUSTYPEn, EIGENSCHAPpen en ROLTYPEn<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/zaaktype                                                                                |
+| Samenhang      | - |
+| Bijzonderheden | -           |
+
+
+
+### Resource: BESLUITTYPE
```

Comment:
Bestaat nog niet in master-versie van ZTC (komt er wel aan binnenkort). Dus even `TODO` bijzetten

Reply:
Besluittype bestaat al wel in de master versie, https://github.com/VNG-Realisatie/gemma-zaken/blob/master/api-specificatie/ztc/openapi.yaml op regel 1298.
Anders had ik het niet in de documentatie opgenomen :-D

Reply:
Lomp van mij :-)